build.gradle.kts

```diff
@@ -45,7 +54,110 @@ subprojects {
     }
 }
 
-repositories {
-    mavenLocal()
-    mavenCentral()
+subprojects {
+    if (name == "exposed-bom") return@subprojects
+
+    apply(plugin = "org.jetbrains.kotlin.jvm")
+
+    testDb("h2") {
+        withContainer = false
+        dialects("H2", "H2_MYSQL", "H2_PSQL", "H2_MARIADB", "H2_ORACLE", "H2_SQLSERVER")
+
+        dependencies {
+            dependency("com.h2database:h2:${Versions.h2_v2}")
+        }
+    }
+
+    testDb("h2_v1") {
+        withContainer = false
+        dialects("H2", "H2_MYSQL")
+
+        dependencies {
+            dependency("com.h2database:h2:${Versions.h2}")
+        }
+    }
+
+    testDb("sqlite") {
+        withContainer = false
+        dialects("sqlite")
+
+        dependencies {
+            dependency("org.xerial:sqlite-jdbc:${Versions.sqlLite3}")
+        }
+    }
+
+    testDb("mysql") {
+        port = 3001
+        dialects("mysql")
+        dependencies {
+            dependency("mysql:mysql-connector-java:${Versions.mysql51}")
+        }
+    }
+
+    testDb("mysql8") {
+        port = 3002
+        dialects("mysql")
+        dependencies {
+            dependency("mysql:mysql-connector-java:${Versions.mysql80}")
+        }
+    }
+
+    testDb("mariadb") {
+        port = 3000
+        dialects("mariadb")
+        dependencies {
+            dependency("org.mariadb.jdbc:mariadb-java-client:${Versions.mariaDB_v2}")
+        }
+    }
+
+    testDb("mariadb_v3") {
+        dialects("mariadb")
+        container = "mariadb"
+        port = 3000
+        dependencies {
+            dependency("org.mariadb.jdbc:mariadb-java-client:${Versions.mariaDB_v3}")
+        }
+    }
+
+    testDb("mariadb_v2") {
+        dialects("mariadb")
+        container = "mariadb"
+        port = 3000
+        dependencies {
+            dependency("org.mariadb.jdbc:mariadb-java-client:${Versions.mariaDB_v2}")
+        }
+    }
+
+    testDb("oracle") {
+        port = 3003
+        dialects("oracle")
+        dependencies {
+            dependency("com.oracle.database.jdbc:ojdbc8:${Versions.oracle12}")
+        }
+    }
+
+    testDb("postgres") {
+        port = 3004
+        dialects("postgresql")
+        dependencies {
+            dependency("org.postgresql:postgresql:${Versions.postgre}")
+        }
+    }
+
+    testDb("postgresNG") {
+        port = 3004
+        dialects("postgresqlng")
+        container = "postgres"
+        dependencies {
+            dependency("org.postgresql:postgresql:${Versions.postgre}")
+            dependency("com.impossibl.pgjdbc-ng:pgjdbc-ng:${Versions.postgreNG}")
+        }
+    }
+
+    testDb("sqlserver") {
+        port = 3005
+        dependencies {
+            dependency("com.microsoft.sqlserver:mssql-jdbc:${Versions.sqlserver}")
+        }
+    }
```

Comment:
`testSqlserver` is the only `verification` gradle task that doesn't work for me. I get the following build fail when trying to run either a single or multiple tests locally:
```
Execution failed for task ':exposed-tests:testSqlserver'.
> No tests found for given includes: [org.jetbrains.exposed.sql.tests.shared.dml.InsertTests](--tests filter)
```

Looking at the most recent branch build on TC, it seems only a handful of test files are having their test cases run on SQL Server.
For example, you can search `Task :exposed-tests:testPostgres` in the build log and it shows that every file in the tree has a plus sign with expandable child tests. Compare this to `Task :exposed-tests:testSqlserver` where only a few test files show test cases that ran.

Reply:
Could you tell me the OS you're using? I will check